listener/listener.go

```diff
@@ -66,3 +76,105 @@ func (l *listener) ProcessEvent(w http.ResponseWriter, request *http.Request) {
 	log.Debug("response payload", zap.Any("response", response.GetOccurrences()))
 	w.WriteHeader(200)
 }
+
+/* TODO need to update this method to use real data */
+func createImagePushOccurrence(eventData *harbor.EventData, repo string) *grafeas_go_proto.Occurrence {
```

Comment:
let's use the vulnerability occurrences for actual vulnerabilities found, and discovery occurrences for events relating to starting and finishing a scan in harbor.